src/get-started/java-binding.md

```diff
@@ -587,15 +604,13 @@ The TestKit also allows creating blocks that contain all current [in-pool][in-po
 transactions:
 
 ```java
-try (TestKit testKit = TestKit.forService(MyServiceModule.class)) {
-  // Put the transaction into the TestKit transaction pool
-  MyService service = testKit.getService(MyService.SERVICE_ID, MyService.class);
+// Instantiate a service that submits a transaction in it's `afterCommit` method
```

Comment:
put that in the description as an example: "in pool txs, e.g., submitted by a service in its `afterCommit` method:"